packages/design-system/src/components/DateField/MultiInputDateField.tsx

```diff
@@ -148,7 +144,7 @@ export function MultiInputDateField(props: DateFieldProps): React.ReactElement {
   });
 
   // Throw away the properties we don't need by destructuring
-  const { id, errorId, ...inputProps } = fieldProps;
+  const { id, ...inputProps } = fieldProps;
```

Comment:
Looks like `errorId` should stay here because `DateInput` doesn't make use of it. Maybe this code would actually be clearer if we used mutation and didn't rename `fieldProps` into `inputProps`. Like if we did

```tsx
delete fieldProps.id;
delete fieldProps.errorId;

...

<DateInput {...fieldProps}
```